crc32c/crc32c.c

```diff
@@ -77,6 +77,10 @@ uint32_t crc32c_software(const uint8_t* buf, uint32_t sz, uint32_t crc)
 
 #ifdef __x86_64__
 
+#include <emmintrin.h>
```

Comment:
Thank you for the patch. `smmintrin.h` is a super set of `emmintrin.h` hence you don't need to `#include <emmintrin.h`.

Reply:
Ah, thanks. Corrected.